tests/GraphQL/SortBlockMutationCreatorTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace DNADesign\Elemental\Tests\GraphQL;
+
+use DNADesign\Elemental\GraphQL\SortBlockMutationCreator;
+use DNADesign\Elemental\Tests\Src\TestElement;
+use GraphQL\Type\Definition\ResolveInfo;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\Security;
+
+class SortBlockMutationCreatorTest extends SapphireTest
+{
+    protected static $fixture_file = 'SortBlockMutationCreatorTest.yml';
+
+    protected static $extra_dataobjects = [
+        TestElement::class
+    ];
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->logInWithPermission('ADMIN');
```

Comment:
It should be logged in as admin by default when you have a fixture file - you can remove this